### DISCUSSION TITLE
Keep the project ID when importing from k1 to k2

### DISCUSSION THREAD

**Discussion by romadon** [kimai/kimai#2537] [On-Premise: Installation & Updates]
Hi all.

I'm importing from K1 to K2, and I need to preserve the projectID from K1, Is there anyway to do this?

Thanks a lot.

**Comment by kevinpapst** [comment]
Not without patching the code and setting the ID manually during the import.
But the original ID is kept as custom field in Kimai 2.

**Reply by romadon** [reply]
Ok, it is good for my, What I have to do? What file I have to modify?

BR.

**Reply by kevinpapst** [reply]
You have to try to pass in the ID somehow:
https://github.com/kevinpapst/kimai2/blob/master/src/Command/KimaiImporterCommand.php#L811
But you are on your own with that, no further support given
